interface/themes/tabs_style_compact.css

```diff
@@ -29,11 +29,12 @@ html
 }
 body
```

Comment:
That's why it looked a bit strange :)
The problem with converting from div to a is that it is breaking the compact view. Would this be an easy css fix?